uilib/widgets/nozzlePreviewWidget.py

```diff
@@ -57,15 +57,23 @@ def loadNozzle(self, nozzle):
             divLen = radDiff / tan(divAngle)
         else:
             divLen = 0
+            
         if convAngle != 0:
             convLen = (outerRad - throatRad) / tan(convAngle)
         else:
             convLen = 0
+            
+        if outerRad <= exitRad:
```

Comment:
Minor nitpick, but you could implement this block with max(). I'll merge if you amend this commit to use that syntax and force push. Functionally, this is exactly what I want!